cmd/crc/cmd/root.go

```diff
@@ -82,8 +85,10 @@ func runRoot() {
 
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
-		logging.Fatal(err)
+		runPostrun()
+		exit.WithMessage(1, err.Error())
```

Comment:
only here we do `exit.WithMessage` ?
it seems like all other instances have been removed... so why keep this?

Reply:
This is the main execute which all commands uses since all command add to the root command so this will going to execute whenever corresponding command have error.

Reply:
I am more question the use of 'exit with message' here, but also perhaps possibility this hides the usage.

See my suggestion to use a generalized wrapper and perhaps don't do this. This is almost like a generic exception handler which can lead to a 'hidden' (hard to find) invocation.

Reply:
Ah got it, right now we are only using this with podman-env side also but I understand your point. I think it is OK to have this as generic handler as of now since there is 2 places where it's happening but in the future might be just have it either part of that generalized wrapper or have following part of `Execute` function.

```
_, _ = output.Fout(os.Stderr, fmt.Sprintf(text, args...))
os.Exit(1)
```

Reply:
I removed `exit.WithMessage`, it's clearer now. 

I also removed podman-env command. It is broken for more than 6 months now.

Reply:
> I also removed podman-env command. It is broken for more than 6 months now.

oops ... waiting for release of V2